### PR TITLE
Pull Request for SAP ILM Store integration

### PR DESCRIPTION
**Pull request by SumitDey2021**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@SumitDey2021 - looks like you still need to sign the CLA before we can accept the contribution...

**Comment by SumitDey2021** [comment]
@bmoore-msft I have now signed the CLA. Could you kindly review the PR and accept the contribution?

**Comment by alex-frankel** [comment]
Re-running the checks since the previous ones were groomed. Will follow up once those are done.

File: application-workloads/sap/sap-ilm-store/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        ],
+        "assignableScopes": [
+          "[concat('/subscriptions/', subscription().subscriptionId, '/resourcegroups/', resourceGroup().name)]"
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.Authorization/roleAssignments",
+      "apiVersion": "2018-07-01",
+      "name": "[guid(resourceGroup().id)]",
```
Comment (bmoore-msft) [inline application-workloads/sap/sap-ilm-store/azuredeploy.json:156]: this needs to be idempotent and use principal, role and scope in the seed.

guid(parameters('azureADAppPrincipalID'), if(equals(parameters('newOrExistingRole'), 'new'), resourceId('Microsoft.Authorization/roleDefinitions/', variables('roleDefinitionId')), resourceId('Microsoft.Authorization/roleDefinitions/', parameters('existingRoleID'))), resourceGroup().id)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: application-workloads/sap/sap-ilm-store/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            ],
+            "dataActions": [
+              "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read",
+              "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write",
+              "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/delete"
+            ]
+          }
+        ],
+        "assignableScopes": [
+          "[concat('/subscriptions/', subscription().subscriptionId, '/resourcegroups/', resourceGroup().name)]"
```
[the remark this answers is not in the document]
Comment (SumitDey2021) [reply application-workloads/sap/sap-ilm-store/azuredeploy.json]: Made the necessary changes

File: application-workloads/sap/sap-ilm-store/azuredeploy.json (hunk 1 above)
Comment (SumitDey2021) [reply application-workloads/sap/sap-ilm-store/azuredeploy.json]: Made the necessary changes

**Pull request merged**